helm-chart/renku-gateway/templates/secret.yaml

```diff
@@ -12,5 +12,5 @@ type: Opaque
 data:
   oidcClientSecret: {{ .Values.oidcClientSecret | default .Values.global.gateway.clientSecret | b64enc | quote }}
   gitlabClientSecret: {{ .Values.gitlabClientSecret | default .Values.global.gateway.gitlabClientSecret | b64enc | quote }}
-  gatewaySecret: {{ required "Fill in .Values.secretKey with `openssl rand -hex 32`"  .Values.secretKey | b64enc | quote }}
```

Comment:
The required flag is actually nice because it makes for more useful error messages. I'd even like to use it for the other secrets in this file too, but afaik this is not possible when you use the `| default` pattern, right?

Reply:
Yeah you cannot have both `required` and `default`. I will switch to having `required` messages everywhere.

Reply:
no, I don't think we can do that. we need the default to be set by the global values.

Reply:
Ok sounds good. I will leave the `default` statements, remove the default secrets I added to the values file and modify the lint command so that it does not fail.